package/com.unity.formats.usd/Runtime/Scripts/Timeline/UsdRecorderBehaviour.cs

```diff
@@ -67,9 +67,9 @@ public class UsdRecorderBehaviour : PlayableBehaviour {
         //  m_usdScene.Stage.SetTimeCodesPerSecond(1);
         //}
 
-        // TODO: How does one extract the time mode (frames or seconds) from the Timeline?
-        Clip.UsdScene.Stage.SetFramesPerSecond(30);
-        Clip.UsdScene.Stage.SetTimeCodesPerSecond(1);
+        Clip.UsdScene.FrameRate = 60;
+        Clip.UsdScene.Stage.SetFramesPerSecond(60);
```

Comment:
Why do you also call SetFramesPerSecond here?
Assigning to FrameRate will set both TimeCodesPerSecond as well as FramesPerSecond